components/org.wso2.carbon.micro.integrator.extensions/org.wso2.carbon.micro.integrator.management.apis/src/main/java/org/wso2/carbon/micro/integrator/management/apis/TaskResource.java

```diff
@@ -148,40 +133,28 @@ private OMElement getTaskByName(MessageContext messageContext, String taskName)
         return null;
     }
 
-    private OMElement convertTaskToOMElement(TaskDescription task) throws XMLStreamException{
+    private JSONObject convertTaskToOMElement(TaskDescription task) throws XMLStreamException{
```

Comment:
`XMLStreamException` is never thrown in this method? Can you please check other IDE warnings as well?

Reply:
Fixed warnings in 5e9e894